code/style.css

```diff
@@ -1,7 +1,194 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+/* styling for all of website*/
 
-.active + .answer {
-  /* some styling here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+  font-family: 'Cormorant', serif;  
+  font-family: 'Cormorant Garamond', serif;
+  font-family: 'Spectral', serif;
 }
+
+/* styling for image-header and it's content*/
+
+.image-header {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  height: 55vw;
+  width: 100vw;
+  background-image: url('hero.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  overflow: hidden;  
+  text-align: center;
+}
+
+.video-header video {
+  position: absolute;
+  left: 0;
+  top: 0;
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+}
+
+.navbar {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: fit-content;
+  top: 0px;
+}
+
+ .logo img {
+  width: 4vw;
+  margin: 5px 20px;
+ }
+
+ul {
+  display: flex;
+  justify-content: space-evenly;
+  width: 100%;
+}
+
+li {
+  list-style: none;
+  color: white; 
+  margin: 1rem;;
+  font-weight: bolder;
+}
+
+.title {
+  position: absolute; 
+  display: flex;
+  flex-direction: column;
```

Comment:
These three lines can be removed from `.title`:
```css 
position: absolute; 
display: flex;
flex-direction: column;
```

`justify-content: center;` is already applied to the `.image-header` div.  Your title will be centered horizontally since it is within that div. Then, in order to center it vertically, you will want `align-self: center;`